libshaderc/include/shaderc.hpp

```diff
@@ -125,6 +126,35 @@ class CompileOptions {
     shaderc_compile_options_set_generate_debug_info(options_);
   }
 
+  // Define an includer interface, on which libshaderc will invoke the methods
+  // to resolve the full path and content of the file to be included, and clean
+  // up the resources used for the including process. Client code should
+  // implement this interface and configure the includer instance then set it to
+  // libshaderc through following api.
+  class Includer {
+   public:
+    virtual shaderc_includer_response* GetIncluderResponse(
```

Comment:
The "Includer" part of these methods is redundant with the class name. You already know you're working with an includer.

I think a more natural naming is:
```
GetInclude
ReleaseInclude
```